code/modules/client/preference_setup/loadout/loadout_gloves.dm

```diff
@@ -98,6 +98,9 @@
 	ringtype["ring, glass"] = /obj/item/clothing/gloves/ring/material/glass
 	ringtype["ring, wood"] = /obj/item/clothing/gloves/ring/material/wood
 	ringtype["ring, plastic"] = /obj/item/clothing/gloves/ring/material/plastic
+	ringtype["ring, uranium"] = /obj/item/clothing/gloves/ring/material/uranium
```

Comment:
If I recall this gives slowly rads and toxin harm...

Reply:
i tested it locally, wore it for solid 30-60 minutes, no changes on healthdoll to show "hey youre dying"